newIDE/app/src/JsExtensionsLoader/BrowserJsExtensionsLoader.js

```diff
@@ -73,6 +73,11 @@ const jsExtensions = [
     extensionModule: require('GDJS-for-web-app-only/Runtime/Extensions/Effects/JsExtension.js'),
     objectsRenderingServiceModules: {},
   },
+  {
+    name: 'P2P Multiplayer',
```

Comment:
This should be the same name as in the extension declaration

Reply:
Also check everywhere that there is no more mention of `Multiplayer` in the extension. This is too low level to be marketed as "multiplayer", I don't want to induce people to error.